manual_cases.md

```
@@ -0,0 +1,533 @@
+| **Version** | 4.17, 4.18, 4.19 |
+| **Assignee** | Ying Zhou (yinzhou) |
+| **Author** | Ying Zhou (yinzhou) |
+| **Importance** | Low |
+| **Test Type** | Non-functional |
+
+#### Test Steps
+
+| # | Step | Expected Result |
+|---|------|-----------------|
+| 1 | with following imagesetconfig and oc-mirror 4.16 v2 <br><br>        <br><br>        kind: ImageSetConfiguration<br><br>        apiVersion: mirror.openshift.io/v2alpha1<br><br>        mirror:<br><br>          operators:<br><br>            - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.14<br><br>        <br><br>        time `oc-mirror --config config.yaml file://out1 --v2` | record the time |
+| 2 | remove the cache directory;<br><br>        with the same imagesetconfig and use oc-mirror 4.17 v2, the default max-parallel-downloads is 80 <br><br>          `time oc-mirror -c config.yaml file://out2 --v2 ` | Should use less time than the first step. |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)